src/module/api/invite/create/index.ts

```diff
@@ -31,13 +31,10 @@ export async function Create(newInvite: ICreateInvite): Promise<any> {
 
   objList.push(obj);
   req.setObjList(objList);
-  console.log("made invite", req.toObject());
 
   const getCreateResponsePb = await new Promise((resolve, reject) => {
     client.create(req, metadata, function (err: any, res: CreateO) {
       if (err) {
-        console.log(err.code);
```

Comment:
These are probably here to ease debugging, but I think it's better if we handle them in the UI and avoid console logs generally. I can revert this if you prefer.

Reply:
Sound good. We take your lead.